src/Microsoft.ML.Api/StatefulFilterTransform.cs

```diff
@@ -4,6 +4,7 @@
 
 using System;
 using System.IO;
+using System.Linq;
```

Comment:
Similar, I guess perhaps you were doing something, then chose to do it a different way?